projects/angular/src/accordion/stepper/stepper.ts

```diff
@@ -32,20 +32,20 @@ import { ClrStepperPanel } from './stepper-panel';
     '[class.clr-accordion]': 'true',
     '[class.clr-stepper-forms]': 'true',
   },
-  providers: [StepperService, { provide: AccordionService, useExisting: StepperService }],
+  providers: [StepperService, { provide: AccordionService, useExisting: StepperService }, ClrDestroyService],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class ClrStepper implements OnInit, OnChanges, AfterViewInit, OnDestroy {
+export class ClrStepper implements OnInit, OnChanges, AfterViewInit {
   @Input('clrInitialStep') initialPanel: string;
   @ContentChildren(ClrStepperPanel, { descendants: true })
   panels: QueryList<ClrStepperPanel>;
-  subscriptions: Subscription[] = [];
   form: FormGroupDirective | NgForm;
 
   constructor(
-    @Optional() private formGroup: FormGroupDirective,
-    @Optional() private ngForm: NgForm,
-    private stepperService: StepperService
+    @Optional() @Inject(FormGroupDirective) private formGroup: FormGroupDirective | null,
```

Comment:
Why do we switch to using `@Inject` here?

Reply:
This is used to explicitly add the `| null` union, since the `formGroup` _might be null_, because of the `@Optional()` flag.

Reply:
I feel like adding `| null` annotations is entirely unrelated and shouldn't be part of this PR/commit.